### ISSUE TITLE
Can't add wallet address for BCH

### ISSUE THREAD

**Issue by mitjax** [nomiddleman/nomiddleman-woocommerce#25]
Hi,
I've installed the plugin, selected a couple of coins and enabled autopay mode.
After selecting BCH I've updated markup to 1% and added wallet address in the form of xpub6.......

I would like to add additional wallets, but after pressing the button "Add more", nothing happens.

Any hints for me?

Thank you.

Br, Mitja

**Comment by mitjax** [comment]
If I use only one address after saving settings, I get "Settings saved". But checking again on BCH shows no configuration was made at all. The error is displayed as:
Bitcoin Cash has no valid wallet addresses. Disabling Bitcoin Cash.
Bitcoin Cash has invalid address: xpub6CQPy....

Wallet address was created at my blockchain.com wallet and copy/pasted here. I can confirm, the length is correct as of 111 characters.

**Comment by rmwb** [comment]
That looks like a Master Public Key (MPK) rather than a BCH address.

**Comment by mitjax** [comment]
You are right. Thank you for pointing me to the right direction. Would you know why can't I add additional Wallet Addresses?

Thanks.
